src/config.ts

```diff
@@ -47,7 +47,7 @@ async function readEdnWorkspaceConfig(uri?: vscode.Uri) {
  * @param {string} data a string representation of a clojure map
  * @returns an error of one was thrown
  */
-async function addEdnConfig(data: string) {
+function addEdnConfig(data: string) {
```

Comment:
We only need to have a function marked async when it awaits on an async function within it.